src/anuncio/infrastructure/anuncio.controller.ts

```diff
@@ -1,13 +1,47 @@
 import { Controller, Get, Param, Body, NotFoundException } from '@nestjs/common';
 import { AnuncioService } from './anuncio.service';
+import { anuncio, reserva, usuario } from '@prisma/client';
+import { GetComentariosDto } from 'src/anuncio/infrastructure/database/dto/get-comentarios.dto';
 import { getAnuncioDto } from './database/dto/get-anuncio.dto';
 import { getUsuarioDto } from './database/dto/get-anuncio-usuario.dto';
 import { getComodidadesAnuncioDto } from './database/dto/get-comodidade-anuncio.dto';
 import { getAnuncioFotosDto } from './database/dto/get-anuncio-fotos.dto';
 
 @Controller('anuncio')
 export class AnuncioController {
-  constructor(private readonly anuncioService: AnuncioService) { }
+  constructor(
+    private readonly anuncioService: AnuncioService,
+  ) {}
+  
+  @Get('comentarios')
+  async getComentarioUser(@Body('id') data: any): Promise<GetComentariosDto[] | object> {
+    return this.anuncioService.getComentarioUser(data);
+  }
+  
+  @Get('media-avaliacao/data-reservas')
+  async getAnuncioHospedeDataMediaAv(@Body('id') id: string): Promise<Object> {
+    return await this.anuncioService.getAnuncioHospedeDataMediaAv(id);
+  }
+
+  @Get('qtd-max-min-diaria')
+  async getQuantMaxEMinDiaria(@Body('id') id: string): Promise<Object> {
+    return await this.anuncioService.getQuantMaxEMinDiaria(id);
+  }
+
+  @Get('politica-cancelamento')
```

Comment:
politica de cancelameto ta vindo 204, preenchimento no banco :
![image](https://github.com/user-attachments/assets/46a8c9a2-41c0-4712-9dbf-dbc532071590)
 e retorno to postman:
![image](https://github.com/user-attachments/assets/4a62a6f9-1fd5-4c1a-afc3-1cf97d098b9f)
rever end point e banco